src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js

```diff
@@ -217,11 +209,6 @@ const transform = (
             reverseField =>
               !reverseField.fieldMainLocation && reverseField.entityPublished,
           )
-          .sort((a, b) =>
-            a.fieldNicknameForThisFacility.localeCompare(
-              b.fieldNicknameForThisFacility,
-            ),
-          )
```

Comment:
Sort by title?

Reply:
I don't think it matters - the sort logic from graphql isn't being applied anywhere, and we usually handle sorting in templates, @mpelzsherman